openedx/core/djangoapps/schedules/apps.py

```diff
@@ -1,12 +1,22 @@
 from django.apps import AppConfig
 from django.utils.translation import ugettext_lazy as _
+from openedx.core.djangoapps.plugins.constants import PluginSignals, ProjectType
 
 
 class SchedulesConfig(AppConfig):
     name = 'openedx.core.djangoapps.schedules'
     verbose_name = _('Schedules')
 
-    plugin_app = {}
+    plugin_app = {
+        PluginSignals.CONFIG: {
+            ProjectType.LMS: {
+                PluginSignals.RECEIVERS: [{
+                    PluginSignals.RECEIVER_FUNC_NAME: u'update_schedules_on_course_start_changed',
+                    PluginSignals.SIGNAL_PATH: u'openedx.core.djangoapps.content.course_overviews.signals:COURSE_START_DATE_CHANGED',
```

Comment:
Maybe it's just me, but needing the `ClassName.` prefixes here really hurts readability, especially because there are so many of them.  I'm a little hesitant to just dump them all in the `constants` module namespace, though.  Any other ideas on how to make this a little easier to read?

Reply:
Would it help if each Enum-class was defined in its own Python module - and we use module encapsulation instead of Class encapsulation?

Reply:
Another idea:
```
from openedx.core.djangoapps.plugins.constants import PluginSignals as ps, ProjectType as pt

    plugin_app = {
        ps.CONFIG: {
            pt.LMS: {
                ps.RECEIVERS: [{
                    ps.RECEIVER_FUNC_NAME: u'update_schedules_on_course_start_changed',
                    ps.SIGNAL_PATH: u'openedx.core.djangoapps.content.course_overviews.signals.COURSE_START_DATE_CHANGED',
                }]
```

Does this read better?